tmt/utils.py

```diff
@@ -2102,3 +2102,31 @@ def update(self, value: str, color: Optional[str] = None) -> None:
 
         sys.stdout.write(f"\r{message}")
         sys.stdout.flush()
+
+
+def find_fmf_root(path: str, just_top: bool = False) -> List[str]:
+    """
+    Search trough path and return all fmf roots that exist there
+
+    If `just_top` is True it will return single item
```

Comment:
```suggestion
    If `just_top` is True it will return list with a single item
```

Reply:
Removing 'just_top' and changing docstring.